src/components/spinner/spinner.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import Spinner from './spinner';
+import notes from './notes.md';
+
+storiesOf('Spinner', module).add('default', () => {
+  const as = select('as', OptionsHelper.colors, Spinner.defaultProps.as);
+  const size = select('size', OptionsHelper.sizesFull, Spinner.defaultProps.size);
+
+  return (
+    <Spinner as={ as } size={ size } />
+  );
+}, { notes: { markdown: notes } });
```

Comment:
Unstyled comments:

![Screenshot 2019-03-13 at 15 09 08](https://user-images.githubusercontent.com/4471057/54289935-fee54400-45a1-11e9-80d8-51a82e16d198.png)

@stevebutler2210 has done a load of work towards styling this content, which has just gone into `master` today. There's a really good example in the mount-in-app storybook component.